packages/grafana-ui/src/utils/processTableData.ts

```diff
@@ -4,6 +4,7 @@ import Papa, { ParseError, ParseMeta } from 'papaparse';
 
 // Types
 import { TableData, Column, TimeSeries } from '../types';
+import { isString } from 'util';
```

Comment:
what is this? can't see a package dependency on util.

Reply:
weird -- not sure, autocomplete from vscode.  I switched to the lodash version.